app/src/main/java/com/vultisig/wallet/ui/models/deposit/DepositFormViewModel.kt

```
@@ -151,7 +158,14 @@ internal data class DepositFormUiModel(
 
     val isAutoCompoundTcyStake: Boolean = false,
     val isAutoCompoundTcyUnStake: Boolean = false,
-)
+
+    val availableSecuredAssets: List<Chain> = listOf(
+    ),
+    val securedAssetWithdrawOptions: List<String> = listOf("Select Secured Asset to Withdraw") + availableSecuredAssets.map { it.ticker() }
+
+) {
+
+}
```

detekt (1.23.8)

[warning] The class or object DepositFormUiModel is empty.

(detekt.empty-blocks.EmptyClassBlock)